routes/index.js

```diff
@@ -142,15 +143,16 @@ module.exports = () => {
 						_profile.bio = account.bio;
 
 						Account.findOne({ username: requestingUser }).then(acc => {
-							if (!acc || !acc.staffRole || acc.staffRole.length === 0 || acc.staffRole === 'contributor') {
-								_profile.lastConnectedIP = 'no looking';
-							} else {
+							if (TRIALMODS.includes(acc) || acc || acc.staffRole || !acc.staffRole.length === 0 || !acc.staffRole === 'contributor') {

```

Comment:
Always true. When acc is false it then checks acc.staffRole which is obv a nullref -hexi

Reply:
but i tested it